blueocean-core-js/src/js/paths/rest.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This object defines rest paths
+ */
+export default {
+    apiRoot() {
+        return '/blue/rest';
+    },
+
+    organizationPipelines(organizationName) {
+        return `${this.apiRoot()}/search/?q=type:pipeline;organization:${encodeURIComponent(organizationName)};excludedFromFlattening:jenkins.branch.MultiBranchProject,hudson.matrix.MatrixProject&filter=no-folders`;
+    },
+
+    allPipelines() {
+        return `${this.apiRoot()}/search/?q=type:pipeline;excludedFromFlattening:jenkins.branch.MultiBranchProject,hudson.matrix.MatrixProject&filter=no-folders`;
+    },
+
+    activities(organization, pipeline) {
+        return `${this.apiRoot()}/organizations/${organization}/pipelines/${pipeline}/activities/`;
+    },
```

Comment:
Right, i see whats happening now. 

This code was copied from existing code. The issue with loading activies actually is in loading the pipeline, because I am generating the href as a key, and the backend is returning a different one. This was never a problem in redux as we were not using it as a key.